packages/DatePicker/src/DatePicker.js

```diff
@@ -169,11 +173,13 @@ function DatePicker(props) {
   }
 
   function handleInputBlur() {
-    window.requestAnimationFrame(() => {
-      if (!isElementContainsFocus(calendarRef.current)) {
-        handleInputConfirm();
-      }
-    });
+    if (calendarRef.current) {
+      window.requestAnimationFrame(() => {
+        if (!isElementContainsFocus(calendarRef.current)) {
```

Comment:
I'm not 100% it's even hypothetically possible, but I was thinking that maybe it's safer to check for the ref within the rAF callback so it's guaranteed not to have changed in between.